### PR TITLE
breaking Convert test into few separate test methods, so failures are easier to diagnose

### PR DESCRIPTION
**Pull request by maumar**

### CODE REVIEW

**Comment by maumar** [comment]
@mikary

File: test/EntityFramework.Core.Tests/EntityFrameworkServiceCollectionExtensionsTest.cs [hunk 1]
```diff
@@ -20,170 +20,170 @@
...
-                        Assert.Same(service, serviceProviderWithCustomService.GetService<TService>());
-                        Assert.Equal(1, serviceProviderWithCustomService.GetServices<TService>().Count());
-                    }
-                }
-            }
-
-            return service;
-        }
-    }
+    //public abstract class EntityFrameworkServiceCollectionExtensionsTest : IDisposable
```
Comment (maumar) [inline test/EntityFramework.Core.Tests/EntityFrameworkServiceCollectionExtensionsTest.cs]: will remove

Comment (maumar) [inline test/EntityFramework.Core.Tests/EntityFrameworkServiceCollectionExtensionsTest.cs]: ignore those, this is temporary due to changes in DI - I will not check this in

**Comment by mikary** [comment]
LGTM :shipit:

**Comment by bricelam** [comment]
Merged as 367d6eb2e3a0a76192e13a4b86b666191a12f7f4

**Pull request closed without merging**